packages/wrangler/src/dev/dev.tsx

```diff
@@ -448,6 +583,25 @@ function DevSession(props: DevSessionProps) {
 				testScheduled: props.testScheduled,
 				getRegisteredWorker: (name) => workerDefinitions[name],
 			},
+			legacy: {
+				site:
```

Comment:
I got this wrong originally, it should be plural: `sites`.

Wonder if it's worth being super-explicit with the product name like `legacy.WorkersSites`